srcpkgs/brother-brscan4/template

```diff
@@ -0,0 +1,75 @@
+# Template file for 'brother-brscan4'
+pkgname=brother-brscan4
+version=0.4.4
+revision=1
+only_for_archs="i686 x86_64"
+depends="sane"
+short_desc="SANE scanner driver for brscan4-compatible printers"
+maintainer="Martijn van Buul <martijn.van.buul@gmail.com>"
+license="custom"
+homepage="http://support.brother.com/"
+repository="nonfree"
+nopie=yes
+_mylibrary="libsane-brother4.so.1.0.7"
+conf_files="/etc/opt/brother/scanner/brscan4/Brsane4.ini /etc/opt/brother/scanner/brscan4/brsanenetdevice4.cfg /etc/opt/brother/scanner/brscan4/models4/*.ini"
+
+if [ "$XBPS_TARGET_MACHINE" = "x86_64" ]; then
+	_debpkgid="4.amd64"
+	distfiles="http://download.brother.com/welcome/dlf006645/brscan4-${version}-${_debpkgid}.deb"
+	checksum=cd255c56e4b10125223315b64eb1205864ab8c759b6f8068c24a076f52bfc577
+	_deblibdir="usr/lib64"
+elif [ "$XBPS_TARGET_MACHINE" = "i686" ]; then
```

Comment:
`else` is enough (the package is only for x86_64 or i686).

Reply:
@pienjo see for reference: https://github.com/void-linux/void-packages/blob/master/srcpkgs/vivaldi/template#L19